### ISSUE TITLE
[BUG] Exhelon Ex3 won’t pair to app for resistance

### ISSUE THREAD

**Issue by JoeyAU2011** [cagnulein/qdomyos-zwift#2081]
I’ve had this bike since early January. Have ridden 185 miles on Zwift and everything has worked perfectly. Connect bike to app and then phone to Zwift via an iPad. Got on 2 days ago and resistance wouldn’t connect on Zwift. Played around with it and noticed I wasn’t able to manually change resistance or gears through the app anymore. Of note my wife uses as well through her phone. She was riding prior to me with no issues. I even repaired through my Zwift profile via her phone and everything worked. So I believe it to be something with my phone/app. 

**To Reproduce**
Turn on bike and attempt to connect to power source, resistance, cadence via Zwift app through QZ app connected to bike. 

**Expected behavior**
Connect to power source, resistance, and cadence to run Zwift effectively.

**Smartphone (please complete the following information):**
 - Device: iPhone 13 Pro max 
 - OS: ios 17.2.1

**Append a debug log**

 Follow this guide https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
  
  
**Additional context**

**Comment by cagnulein** [comment]
Hi @JoeyAU2011 please attach a debug log

**Comment by JoeyAU2011** [comment]
Shoot sorry.

Joey Tatara

**Comment by JoeyAU2011** [comment]
[debug-Wed_Jan_31_12_16_13_2024.log](https://github.com/cagnulein/qdomyos-zwift/files/14115710/debug-Wed_Jan_31_12_16_13_2024.log)

**Comment by JoeyAU2011** [comment]
Done

Joey Tatara

**Comment by cagnulein** [comment]
In this log I saw that you asked changed resistance to 2 and then 1 and bike answered correctly changing resistance. Am I missing something?

**Comment by JoeyAU2011** [comment]
Correct. It has always paired and seems to be what indicates control of the
auto resistance feature as far as I can tell. I’ve always been able to
connect to it and it has worked properly and now I cannot and it is not.

If I change gears or resistance through QZ it does nothing now as well. I
never touch resistance but obviously to use gears.

Joey Tatara

**Comment by cagnulein** [comment]
Try to reset the network setting of the device with zwift and reboot it

It sounds like a Bluetooth cache issue due to an ios update

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 31 gen 2024 alle 19:50 JoeyAU2011 ***@***.***>
ha scritto:

**Comment by JoeyAU2011** [comment]
I ran iOS update on both my phone and the iPad that runs Zwift, then
disconnected and reconnected each device from internet, and unpaired each
device from each other in Bluetooth.

It works now! Boom 💥

Joey Tatara

**Comment by cagnulein** [comment]
Glad it worked! Feel free to open this again if you still get the issue
